README.md

```diff
@@ -367,4 +367,22 @@ The following flags are available for `patch subscriptions`:
 
 * Patch a subscription by its ID and change the status to 'Reserved' (dry run) `echo '{ "status": "Reserved" }' | ocm support patch subscriptions [subID]`
 * Patch a subscription and change the support level to Self-Support for subscriptions with 'Reserved' status (no dry run) `echo '{ "support_level": "Self-Support" }' | ocm support patch subs --filter "status='Reserved' --dryRun=false`
-* Patch all subscriptions of an organization and change the status to Archived (no dry run and set maxRecords more than the actual number of affected records) `echo '{ "status": "Archived" }' | ocm support patch subs --filter "organization_id='[orgID]' --dryRun=false --maxRecords=1000`
\ No newline at end of file
+* Patch all subscriptions of an organization and change the status to Archived (no dry run and set maxRecords more than the actual number of affected records) `echo '{ "status": "Archived" }' | ocm support patch subs --filter "organization_id='[orgID]' --dryRun=false --maxRecords=1000`
+
+#### Patching organizations
+
+Use the `organizations` subcommand to patch an organization by passing the ID, or provide filter value to search matching organizations and patch them. Pass the JSON body for the patch request in terminal using `echo '{<PATCH_BODY>}' | ` before the actual command. By default, the dry run flag will be enabled. Pass `dryRun=false` flag to actually remove the resource.
+
+The following flags are available for `patch organizations`:
+
+```
+--filter                     If non-empty, filters and patches the matching organizations.
+--dryRun                     If false, it will execute the patch command call in instead of a dry run.
+--maxRecords                 Maximum number of affected records. Defaults to 100. Only effective when dryRun is set to false.
+-h, --help                   help for patch
+```
+
+##### Examples
+
+* Patch an organization by its ID and change the externalID (dry run) `echo '{ "external_id": "12541229" }' | ocm support patch organizations [orgID]`
+* Patch all organizations with names starting with "Red Hat" and change the name to "Red Hat Inc." (no dry run and set maxRecords more than the actual number of affected records) `echo '{ "name": "Red Hat Inc." }' | ocm support patch orgs --filter "name like 'Red Hat%' --dryRun=false --maxRecords=1000`
```

Comment:
Does it make sense to split this in two commands?

1) ocm support patch organization [orgID]

2) ocm support patch organizations [Filter]

This way we could drop the filter flag, making it easier for users to understand.

Reply:
That makes sense, but would it be complex for users to understand? A single command could be simpler

Reply:
Once the users understand this logic (singular vs plural resources), we can apply to other commands - for example, patch subscriptions, get account (vs get accounts), etc.

Also, the OCM CLI already does something like that. For example:

ocm get organizations
ocm get organization - expects and ID as argument.